### PR TITLE
Feature/api base

### PR DESCRIPTION
**Pull request by michalszc**

### CODE REVIEW

File: backend/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.env
```
Comment (Tosiekdev) [inline backend/.gitignore]: newline?

File: backend/api/resource/users/handler.go [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+	"net/http"
+
+	"github.com/go-chi/chi"
+	"github.com/go-playground/validator/v10"
+	"github.com/google/uuid"
+	"github.com/rs/zerolog"
+	"gorm.io/gorm"
+
+	e "backend/api/resource/common/error"
+	validatorUtil "backend/utils/validator"
```
Comment (Tosiekdev) [inline backend/api/resource/users/handler.go:15]: why alias?

File: backend/api/resource/users/handler.go [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+		repository: NewRepository(db),
+		validator:  v,
+		logger:     l,
+	}
+}
+
+// List godoc
+//
+//	@summary		List users
+//	@description	List users
```
Comment (Tosiekdev) [inline backend/api/resource/users/handler.go:35]: not pretty formated

File: backend/docs/swagger.json [hunk 4]
```diff
@@ -0,0 +1,324 @@
...
+                "id": {
+                    "type": "string"
+                },
+                "name": {
+                    "type": "string"
+                }
+            }
+        }
+    }
+}
```
Comment (Tosiekdev) [inline backend/docs/swagger.json:324]: newline?

**Review by Tosiekdev — CHANGES_REQUESTED** [review]
Overall good, but has some minor issues

File: backend/api/resource/users/handler.go (hunk 2 above)
Comment (michalszc) [reply backend/api/resource/users/handler.go:15]: due to the fact that other package import validator, so i don't want naming conflicts

File: backend/docs/swagger.json (hunk 4 above)
Comment (michalszc) [reply backend/docs/swagger.json:324]: All files in docs are generated by swag command.

**Review by Tosiekdev — APPROVED** [review]
good

**Pull request merged**